### PR TITLE
Issue template: add prompts for versions

### PR DESCRIPTION
**Pull request by hugovk**
Changes proposed in this pull request:

 * Add OS version to the issue template
 * Prompt for it and Python and Pillow versions
 * This might encourage people to fill in the info in bug reports

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -4,7 +4,11 @@
 
 ### What actually happened?
 
-### What versions of Pillow and Python are you using?
+### What versions of OS, Python and Pillow are you using?
```
Comment (radarhere) [inline .github/ISSUE_TEMPLATE.md]: I might be nitpicking, but the wording of this is clunky to me. An alternate suggestion - 'What are the versions of your OS, Python and Pillow?'

Comment (hugovk) [reply .github/ISSUE_TEMPLATE.md]: Yes, that's an improvement.

Which do you prefer, that or

> What are your OS, Python and Pillow versions?

Comment (radarhere) [reply .github/ISSUE_TEMPLATE.md]: Either sounds fine.

Comment (hugovk) [reply .github/ISSUE_TEMPLATE.md]: Thanks, updated with the shorter one.

**Comment by radarhere** [comment]
Thanks

**Pull request merged**